service/administration/service.go

```diff
@@ -0,0 +1,50 @@
+package administration
+
+import (
+	"context"
+
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/client"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/session"
+)
+
+// Service provides the API operation methods for making requests to endpoints
+// of the Spotinst API. See this package's package overview docs for details on
+// the service.
+type Service interface {
+	ListUsers(context.Context, *ListUsersInput) (*ListUsersOutput, error)
+	CreateUser(context.Context, *User, *bool) (*CreateUserOutput, error)
+	CreateProgUser(context.Context, *ProgrammaticUser) (*CreateProgrammaticUserOutput, error)
+	ReadUser(context.Context, *ReadUserInput) (*ReadUserOutput, error)
+	ReadProgUser(context.Context, *ReadUserInput) (*ReadProgUserOutput, error)
```

Comment:
Is this a separate API, I couldn't find.

Reply:
the route is same but the response is slightly different so handling with a different function